src/js/components/core/SideBar/StatusBar.js

```diff
@@ -65,23 +64,21 @@ class StatusBar extends React.Component{
     if(this.state.currentCheckNamespace !== ""){
       groups = api.getDataFromCheckStore(this.state.currentCheckNamespace, 'groups');
     }
-    console.log(groups);
-  if(groups){
-    console.log("it is doing it..");
-    var currentGroupIndex = api.getDataFromCheckStore(this.state.currentCheckNamespace, 'currentGroupIndex');
-    console.log(currentGroupIndex);
-    var currentCheckIndex = api.getDataFromCheckStore(this.state.currentCheckNamespace, 'currentCheckIndex');
-    console.log(currentCheckIndex);
-    var currentCheck = groups[currentGroupIndex]['checks'][currentCheckIndex];
-    if(this.state.currentCheckNamespace === "TranslationWordsChecker"){
-      this.setState({currentWordOrPhrase: currentCheck.word});
-    }else if (this.state.currentCheckNamespace === "TranslationNotesChecker") {
-      this.setState({currentWordOrPhrase: currentCheck.group});
-    }else {
-      console.error("currentCheckNamespace is undefined in CoreStore");
+    if(groups){
+      var currentGroupIndex = api.getDataFromCheckStore(this.state.currentCheckNamespace, 'currentGroupIndex');
+      var currentCheckIndex = api.getDataFromCheckStore(this.state.currentCheckNamespace, 'currentCheckIndex');
+      var currentCheck = groups[currentGroupIndex]['checks'][currentCheckIndex];
+      if(this.state.currentCheckNamespace === "TranslationWordsChecker"){
+        this.setState({currentWordOrPhrase: currentCheck.word});
```

Comment:
We need to make this process generic, for the sake of future devs working on tC tools we shouldnt need to add code to our statusbar for them to start developing their app. The most apparent solution would be defining a common convention for what to call your current check. `currentCheck.menuTitle` comes to mind but it could be anything. This is just poor practice for sustainability.